pycypher/local_tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright 2017, Google Inc. All rights reserved.
```

Comment:
Google?

Reply:
Yes, this is on company time. All the paperwork is already set up, no need to worry.

Reply:
Ok. Perhaps create an AUTHORS file in the root to give context?

Reply:
I don't think this is necessary. I think that the notices on top of files clearly specify copyright holders and `git log` is enough for attribution purposes.

Reply:
Ok. Speaking of paperwork, I should probably get a CLA agreement signed to make sure there's no copyright holder issues later. I do that already for libneo4j-parser, but I haven't done so for this project yet (as I was the only contributor until now).

Here's the CLA link: https://www.clahub.com/agreements/cleishm/libcypher-parser

Does this work for you/google?

Reply:
Well, good to know. I assumed that there is no CLA for this project since it is not mentioned anywhere in the Readme and project page. Currently I have an approval for patching a project without a CLA.

Reply:
I just created the CLA, since there wasn't one. Check if you have approval for that, and we can chat if otherwise.

Reply:
(To be clear, I'm open to being convinced we don't need one for this project!)

Reply:
I applied for approval, we will see what will happen.

Reply:
Hey @franekp. How did that approval go? I did some research of my own, and I also think we could skip it for this sort of project. The main concern needing a CLA is ensuring that it's easy to adjust licensing and copyright later. In this case, I don't see that being as big an issue. What do you think?

Reply:
I got an approval, so I can sign the CLA.